analysis/priestdiscipline/src/CONFIG.tsx

```diff
@@ -7,10 +7,10 @@ import CHANGELOG from './CHANGELOG';
 
 const config: Config = {
   // The people that have contributed to this spec recently. People don't have to sign up to be long-time maintainers to be included in this list. If someone built a large part of the spec or contributed something recently to that spec, they can be added to the contributors list. If someone goes MIA, they may be removed after major changes or during a new expansion.
-  contributors: [Khadaj, Oratio],
+  contributors: [Khadaj, Oratio, Reglitch],
   // The WoW client patch this spec was last updated.
-  patchCompatibility: null,
-  isPartial: true,
+  patchCompatibility: '9.0.5',
```

Comment:
Spec is around 90% of the way there, it's definitely at the point you can use it to analyze choices. More stuff on the way!